pkg/plugins/manager/fakes/fakes.go

```diff
@@ -575,6 +578,7 @@ func (p *FakeBackendPlugin) IsManaged() bool {
 func (p *FakeBackendPlugin) Exited() bool {
 	p.mutex.RLock()
 	defer p.mutex.RUnlock()
+	go func() { p.ExitedCheckDone <- struct{}{} }()
```

Comment:
Does this need to be here too? 🤔

Reply:
yeah, the happy path will execute this so we ensure that the ticker has been executed but the plugin has not been stopped.

The other execution (line 555) only ensures a fast failure if the plugin is indeed stopped.

Reply:
Ah in that case may be worth adding a comment to line 555. Based on the field name it makes sense in the func, but not so much in `Stop`